app/models/company.rb

```diff
@@ -1,2 +1,7 @@
 class Company < ActiveRecord::Base
+  validates :name,            presence: true
+  validates :description,     presence: true
+  validates :sub_description, presence: true
```

Comment:
![screenshot from 2015-11-06 07 38 51](https://cloud.githubusercontent.com/assets/5815800/10989540/811d3ed6-8459-11e5-97d2-5106db617168.png)

@arafatm From that :point_up: 
- 3SIXD CONSULTING = description of the company 3sixD
- We create openings ... = sub description
- 3sixD = company name, it can be any like Furaha or Google